test/test_crud.py

```diff
@@ -2,7 +2,12 @@
 from ansible.cli.playbook import PlaybookCLI
 
 
-MODULES = ['organization', 'product']
+MODULES = [
+    'organization',
+    'product',
+    'content_view',
+    'composite_content_view'
+]
 
```

Comment:
While we are at it, let's get them in alphabetical order.
This list is gonna be long. ;)

Reply:
i almost did.. will do